README.md

```
@@ -42,132 +42,59 @@ WillMeter は意思力を「見える化」することで：
 - **開発環境**: Xcode 16.4+
-- **アーキテクチャ**: DDD (Domain-Driven Design) + TDD (Test-Driven Development)
-- **品質管理**: SwiftLint + Code Rabbit AI Review
+- **アーキテクチャ**: Simple First（軽量・シンプル優先） + TDD (Test-Driven Development)
+- **品質管理**: SwiftLint + AI自動レビュー（[docs/ai-review.md](docs/ai-review.md)）
 
 ## 🏗️ アーキテクチャ
 
-本プロジェクトは **Clean Architecture + DDD** を採用し、4層構造でレイヤー分離を実現：
+本プロジェクトは **Simple First** を設計方針とします。ドメインロジックを UI・永続化から分離しつつ、レイヤーや抽象化の一律強制はしません。判断の経緯は [ADR 0002](docs/adr/0002-adopt-simple-first-architecture.md) を参照してください。
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)